app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -290,4 +326,68 @@ public void finalize() throws Throwable {
     }
+
+    //inserts values to the multiple choice card table
+    public void insertMCCard(MultipleChoiceCard mcCard, SQLiteDatabase db) {
+        ContentValues cv = new ContentValues();
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_QUESTION, mcCard.getQuestion());
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION1, mcCard.getOption1());
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION2, mcCard.getOption2());
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION3, mcCard.getOption3());
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION4, mcCard.getOption4());
+        cv.put(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_ANSWER, mcCard.getAnswer());
+        long id;
+        id = db.insert(MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME, null, cv);
+        mcCard.setId(id);
+    }
+
+    public void deleteMCCard(MultipleChoiceCard mcCard) {
+        db.delete(MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,
+                "id" + "=" + mcCard.getId(),
+                null);
+    }
+
+    public List<MultipleChoiceCard> getAllMultipleChoiceCards() {
+        List<MultipleChoiceCard> mcList = new ArrayList<>();
+
+        db = getReadableDatabase();
+        Cursor c = db.rawQuery("SELECT * FROM " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME, null);
+        addToMCList(c, mcList);
+        c.close();
+
+        return mcList;
+    }
+
+    private void addToMCList(Cursor c, List<MultipleChoiceCard> list) {
+        if (c.moveToFirst()) {
+            do {
+                MultipleChoiceCard mcCard = new MultipleChoiceCard();
+                mcCard.setId((c.getLong(c.getColumnIndex("id"))));
+                mcCard.setQuestion((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_QUESTION))));
+                mcCard.setOption1((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION1))));
+                mcCard.setOption2((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION2))));
+                mcCard.setOption3((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION3))));
+                mcCard.setOption4((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_OPTION4))));
+                mcCard.setAnswer((c.getString(c.getColumnIndex(MultipleChoiceContract.MultipleChoiceCardTable.COLUMN_ANSWER))));
+
+                list.add(mcCard);
+
+            } while (c.moveToNext());
+        }
+    }
+
+    public void updateMCId(String newId, String oldId) {
+        db.execSQL("update " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME +" set id = " + newId + " where id = " + oldId);
+
+    }
+
+    //test data for multiple choice table
+    public void fillMCTable(SQLiteDatabase db) {
```

Comment:
fillMCTable --> fillMultipleChoiceTable

Can you move this to the tests file since we are only using this function for testing?